build/BuildScript.cs

```diff
@@ -51,10 +51,12 @@ protected override void ConfigureTargets(ITaskContext context)
                 .SetAsDefault()
                 .DependsOn(build, runTests);
             
+            var branch = Environment.GetEnvironmentVariable("APPVEYOR_REPO_BRANCH");
             context.CreateTarget("Rebuild.Server")
                 .SetAsDefault()
                 .DependsOn(rebuild)
-                .DependsOn(nugetPublish);
+                .DependsOn(nugetPublish)
+                   .When((c) => c.BuildSystems().RunningOn == BuildSystemType.AppVeyor && branch != null && branch.Equals("master", StringComparison.OrdinalIgnoreCase));
```

Comment:
AppVeyor will run in both Windows and Linux. This seems will publish twice?

Reply:
No it won't If you look in appveyor yaml you will see that:
- build on windows run's target Rebuild.Server.
- build on linux run's target Rebuild which does not publishes nuget packages.

But even if nuget packages would be published on both environments one would silently fail because nuget package with same version would already exists.

Reply:
It sounds good!